sys-fs/jfsutils/files/jfsutils-1.1.15-musl-fix-includes.patch

```diff
@@ -0,0 +1,18 @@
+--- jfsutils-1.1.15.orig/libfs/fssubs.c
```

Comment:
1. Always include a link to patch sources.
2. Include a link to any relevant Gentoo bugs (or others).
3. Change the patch path to be cleaner (--- a/ and +++ b/)

Reply:
This should be all the changes you asked for however I am unable to find any other bugs related to this issue other than the one included on b.g.o.

Reply:
that's ok, but please put the info at the top of the patch